tests/general_generators.py

```diff
@@ -66,6 +68,23 @@ def any_file_contents(byte_count: int = 10) -> bytes:
     return urandom(byte_count)
 
 
+def any_request_id() -> str:
+    """Arbitrary-length string"""
+    return uuid4().hex
+
+
+def any_http_status_code() -> int:
+    http_statuses = [
+        status for status in dir(HTTPStatus) if not callable(getattr(HTTPStatus, status))
+    ]
+    return int(getattr(HTTPStatus, choice(http_statuses)))
```

Comment:
Surely there is a more elegant way to write this...

Reply:
`return choice(list(HTTPStatus))` works.

Reply:
`return choice(list(HTTPStatus))` [works](https://stackoverflow.com/a/24243545/96588).